docs/source/WritingTests.rst

```diff
@@ -520,6 +520,11 @@ If you need to perform setup actions before/after your test, you may do so
 in the ``setUp`` and ``tearDown`` methods, respectively. We'll give examples
 in the following section.
 
+The ``teadDown`` method is always executed (unless the test is skipped).
```

Comment:
Actually I'd rather re-phrase it in order to make sense as it talks about next section and then you add another sentence which makes at least me a bit confused. How about:

_To perform setup actions before/after your test, you may use ``setUp`` and ``tearDown`` methods. The ``tearDown`` method is always executed even on ``setUp`` failure so don't forget to initialize your variables early in the ``setUp``. Example of usage is in the next section :doc:`Running third party...`_